From 61aea32efe675e584121295452fc7ab2a652d021 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Wed, 20 Apr 2016 09:39:30 -0300
Subject: [PATCH] wscript: don't check for fork() on POSIX checks

It's not being used, and in fact prevents mpv from being built in nommu
architectures (cortex M, blackfin, ...)

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
Status: pull request https://github.com/mpv-player/mpv/pull/3068

 wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wscript b/wscript
index a4ffce6..40328e4 100644
--- a/wscript
+++ b/wscript
@@ -127,7 +127,7 @@ main_dependencies = [
         'desc': 'POSIX environment',
         # This should be good enough.
         'func': check_statement(['poll.h', 'unistd.h', 'sys/mman.h'],
-            'struct pollfd pfd; poll(&pfd, 1, 0); fork(); int f[2]; pipe(f); munmap(f,0)'),
+            'struct pollfd pfd; poll(&pfd, 1, 0); int f[2]; pipe(f); munmap(f,0)'),
     }, {
         'name': 'posix-or-mingw',
         'desc': 'development environment',
-- 
2.7.3

